test/PhpInputStreamTest.php

```diff
@@ -64,6 +64,17 @@ public function testGetContentsReturnsRemainingContentsOfStream()
         $this->assertEquals(substr($contents, 128), $remainder);
     }
 
+    public function testGetContentsReturnCacheWhenReachedEof()
+    {
+        $this->stream->getContents();
+        $this->assertStreamContents($this->stream->getContents());
+
+        $stream = new PhpInputStream('data://,0');
```

Comment:
I intended just represent `string(1) "0"` for PHP's stream. ( http://php.net/wrappers.data )